scss/mixins/_screen-reader.scss

```diff
@@ -10,6 +11,8 @@
   margin: -1px;
```

Comment:
Can we drop the `margin: -1px`? I don't see it in Hugo's post and I know we had an issue with it on GitHub causing a horizontal scrollbar with that.